ccm_web/client/src/utils/fileUtils.ts

```diff
@@ -0,0 +1,8 @@
+function createOutputFileName (file: File, newfileEnding: string): string {
+  const splitName = file.name.split('.')
+  const filenameIndex = splitName.length >= 2 ? splitName.length - 2 : 0
+  splitName[filenameIndex] = splitName[filenameIndex] + newfileEnding
+  return splitName.join('.')
```

Comment:
Few sentences what those numbering login means. Otherwise somebody need to debug of console log what's going on

Reply:
This was taken from `GradebookCanvas` where Chris had implemented it (no description there). I can try to describe it though.

Reply:
Done. https://github.com/tl-its-umich-edu/canvas-course-manager-next/pull/223/commits/99e22b927f0354fe8fd05063801c11de281fd6c0

Reply:
Thanks @ssciolla. I often noticed myself if coming back to project after a gap I don't not remember what particular might be doing. comments are good.

But certain Practitioner don't like comments in code but instead advocate for descriptive names. Just a matter of preference